### PR TITLE
Db/win omnibus packaging

### PR DESCRIPTION
**Pull request by derekwbrown**
Match windows omnibus packaging strategy to linux.
Requires https://github.com/DataDog/omnibus-ruby

### CODE REVIEW

**Review by truthbk — CHANGES_REQUESTED** [review]
Added some feedback

File: omnibus/Gemfile [hunk 1]
```diff
@@ -2,9 +2,10 @@ source 'https://rubygems.org'
 
 # Install omnibus
 # TODO: allow to specify the branch as an env var
-gem 'omnibus', git: 'https://github.com/DataDog/omnibus-ruby.git', branch: 'datadog-5.5.0'
+gem 'omnibus', git: 'https://github.com/DataDog/omnibus-ruby.git', branch: 'db/generic_extraconfs'
```
[the remark this answers is not in the document]
Comment (derekwbrown) [reply omnibus/Gemfile]: No, has to be changed back.  But, won't pass CI (and therefore won't be allowed to merge) w/o that, until the companion PR  https://github.com/DataDog/omnibus-ruby/pull/47 is pushed.

File: omnibus/Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,95 @@
+GIT
```
[the remark this answers is not in the document]
Comment (derekwbrown) [reply omnibus/Gemfile.lock]: wasn't intentional; I'll back it out.

File: omnibus/config/software/datadog-agent-finalize.rb [hunk 3]
```diff
@@ -13,29 +13,45 @@
...
-    # Move the deprecation placeholder
-    move "#{install_dir}/bin/agent/dd-agent", "/usr/bin/dd-agent"
+if windows?
+    build do
+        conf_dir_root = "#{Omnibus::Config.source_dir()}/etc/datadog-agent"
+        conf_dir = "#{conf_dir_root}/extra_package_files/EXAMPLECONFSLOCATION"
+        mkdir conf_dir
+        move "#{install_dir}/etc/datadog-agent/datadog.yaml.example", conf_dir_root, :force=>true
+        move "#{install_dir}/etc/datadog-agent/trace-agent.conf", conf_dir_root, :force=>true
+        #move "#{install_dir}/etc/datadog-agent/process-agent.conf", conf_dir
```
[the remark this answers is not in the document]
Comment (derekwbrown) [reply omnibus/config/software/datadog-agent-finalize.rb]: Process agent will not ship with windows initially; left this here as a placeholder for when it is added.

File: omnibus/config/projects/agent.rb [hunk 4]
```diff
@@ -87,14 +88,16 @@
   bundle_theme true
   wix_candle_extension 'WixUtilExtension'
   wix_light_extension 'WixUtilExtension'
+  extra_package_files "#{Omnibus::Config.source_dir()}\\etc\\datadog-agent\\extra_package_files"
```
[the remark this answers is not in the document]
Comment (derekwbrown) [reply omnibus/config/projects/agent.rb]: I may have named `extra_package_files` poorly, given the existence of extra package `extra_package_file`.  They do provide a similar functionality, though.  Makes much more sense when you consider https://github.com/DataDog/omnibus-ruby/pull/47

**Review by truthbk — APPROVED** [review]
:shipit:

**Pull request merged**